lib/Rx/Observable/ErrorObservable.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Rx\Observable;
+
+use Exception;
+use Rx\Disposable\EmptyDisposable;
+
+class ErrorObservable extends BaseObservable
```

Comment:
@asm89 As far as I can tell, `ErrorObservable` and `ThrowObservable` are the same.  Should we get rid of one?